src/diff.js

```diff
@@ -181,6 +181,7 @@ export function circular() {
 	return function print(key, val) {
 		if (val === void 0) return '[__VOID__]';
 		if (typeof val === 'number' && val !== val) return '[__NAN__]';
+		if (Object.prototype.toString.call(val) === '[object BigInt]') return val.toString();
```

Comment:
Are `Object(BigInt(3))` and `Object(4n)` really something to lookout for?

```suggestion
		if (typeof val === 'bigint') return val.toString();
```

Reply:
I'm not an avid user of `BigInt` so my honest answer is "not sure!" 😅 I bet it's the same kind of deal as `new String('x')` having a `typeof` that returns `object`. Exceedingly rare but technically possible. I don't think it'd functionally be a problem to simplify it to this.

Reply:
I think we should go with the `typeof` check & if someone comes forward with the need for `[object BigInt]` then you both can "told you so" me :D

Reply:
Hah, fair enough. 😄 I'm sure this will be fine!